attestation-agent/attestation-agent/src/bin/grpc-aa/main.rs

```diff
@@ -45,6 +54,23 @@ pub async fn main() -> Result<()> {
     let attestation_socket = cli.attestation_sock.parse::<SocketAddr>()?;
 
     let mut aa = AttestationAgent::new(cli.config_file.as_deref()).context("start AA")?;
+    if let Some(initdata) = cli.initdata {
+        info!("Initdata is given by parameter, try to check.");
+        let initdata = base64::engine::general_purpose::STANDARD
+            .decode(&initdata)
+            .context("base64 decode initdata")?;
+        let res = aa
+            .bind_init_data(&initdata)
+            .await
+            .context("check initdata")?;
```

Comment:
This where the AA will crash if the initdata check does not match, right?

Reply:
Thanks. Added the more expressive language.